From: Agathe Porte <agathe.porte@oss.qualcomm.com>
Subject: [PATCH] Remove exec flag and shebangs from sourced scripts
Date: Thu, 18 Jun 2026 15:34:39 +0200

Drop the executable flag and shebangs from bash scripts that are only
sourced from other scripts and have no use of being executed.

This makes the `script-not-executable` and `executable-in-usr-lib`
lintian tags disappear.

Signed-off-by: Agathe Porte <agathe.porte@oss.qualcomm.com>
Forwarded: https://github.com/qualcomm-linux/pkg-qcom-adreno/issues/38

---
 .../arm64/usr/lib/qcom-adreno-common/qcom-adreno-common-vars.sh | 2 --
 .../arm64/etc/profile.d/qcom-adreno-vulkan-profile.sh           | 2 --
 .../arm64/usr/lib/qcom-adreno-vulkan/qcom-adreno-vulkan-vars.sh | 2 --
 3 files changed, 6 deletions(-)
 mode change 100755 => 100644 data/qcom-adreno-common1/arm64/usr/lib/qcom-adreno-common/qcom-adreno-common-vars.sh

diff --git a/data/qcom-adreno-common1/arm64/usr/lib/qcom-adreno-common/qcom-adreno-common-vars.sh b/data/qcom-adreno-common1/arm64/usr/lib/qcom-adreno-common/qcom-adreno-common-vars.sh
old mode 100755
new mode 100644
index 15060f5..890e310
--- a/data/qcom-adreno-common1/arm64/usr/lib/qcom-adreno-common/qcom-adreno-common-vars.sh
+++ b/data/qcom-adreno-common1/arm64/usr/lib/qcom-adreno-common/qcom-adreno-common-vars.sh
@@ -1,3 +1 @@
-#!/bin/sh
-
 export QT_QPA_PLATFORM=wayland
diff --git a/data/qcom-adreno-vulkan1/arm64/etc/profile.d/qcom-adreno-vulkan-profile.sh b/data/qcom-adreno-vulkan1/arm64/etc/profile.d/qcom-adreno-vulkan-profile.sh
index 2b31366..7821314 100644
--- a/data/qcom-adreno-vulkan1/arm64/etc/profile.d/qcom-adreno-vulkan-profile.sh
+++ b/data/qcom-adreno-vulkan1/arm64/etc/profile.d/qcom-adreno-vulkan-profile.sh
@@ -1,5 +1,3 @@
-#!/bin/sh
-
 if [ -f /usr/lib/qcom-adreno-vulkan/qcom-adreno-vulkan-vars.sh ]; then
     source /usr/lib/qcom-adreno-vulkan/qcom-adreno-vulkan-vars.sh
 fi
diff --git a/data/qcom-adreno-vulkan1/arm64/usr/lib/qcom-adreno-vulkan/qcom-adreno-vulkan-vars.sh b/data/qcom-adreno-vulkan1/arm64/usr/lib/qcom-adreno-vulkan/qcom-adreno-vulkan-vars.sh
index faa3df4..b9628a6 100644
--- a/data/qcom-adreno-vulkan1/arm64/usr/lib/qcom-adreno-vulkan/qcom-adreno-vulkan-vars.sh
+++ b/data/qcom-adreno-vulkan1/arm64/usr/lib/qcom-adreno-vulkan/qcom-adreno-vulkan-vars.sh
@@ -1,3 +1 @@
-#!/bin/sh
-
 export VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/adrenovk.json
-- 
2.53.0

